theanomodels/models/mlp.py

```diff
@@ -9,9 +9,9 @@
 from theano.printing import pydotprint
 import theano.tensor as T
 from theano.sandbox.rng_mrg import MRG_RandomStreams as RandomStreams
-from utils.optimizer import adam,rmsprop
-from utils.misc import saveHDF5
-from __init__ import BaseModel
+from ..utils.optimizer import adam,rmsprop
```

Comment:
if you are outside theanomodels, then do from `theanomodels.utils.misc import XYZ`